nilearn/maskers/tests/conftest.py

```diff
@@ -17,3 +17,27 @@ def data_1(shape_3d_default):
 def mask_img_1(data_1, affine_eye):
     """Return a mask image."""
     return Nifti1Image(data_1.astype("uint8"), affine_eye)
+
+
+@pytest.fixture
+def shape_mask():
+    """Shape for masks."""
+    return (13, 14, 15)
+
+
+@pytest.fixture
+def shape_maps():
+    """Shape for maps."""
+    return (16, 17, 18)
+
+
+@pytest.fixture
+def length():
+    """Return a default length for 4D images."""
+    return 10
+
+
+@pytest.fixture
+def n_regions():
+    """Return a default numher of regions for maps."""
+    return 9
```

Comment:
```suggestion
    return 9
    
@pytest.fixture
def maps11_img(shape_3d_default, n_regions, affine_eye):
    return generate_maps(
        shape=shape_3d_default,
        n_regions=n_regions,
        affine=affine_eye
    )[0]


@pytest.fixture
def fake_mri11(shape_3d_default, affine_eye, length):
    return generate_fake_fmri(
        shape_3d_default, affine=affine_eye, length=length
    )
```

Reply:
What do you think about adding these fixtures?

Reply:
yup: thanks a bunch for checking those